spanner/changestreams/src/test/java/com/example/spanner/changestreams/ChangeStreamSampleIT.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.spanner.changestreams;
+
+import static org.junit.Assert.assertNotNull;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+ * Tests for ChangeStreamSample.
+ */
+@RunWith(JUnit4.class)
+@SuppressWarnings("checkstyle:abbreviationaswordinname")
+public class ChangeStreamSampleIT {
+  private static String instanceId = System.getProperty("spanner.test.instance");
+  private static String databaseId = System.getProperty("spanner.test.database");
+  private static String prefix = System.getProperty("spanner.test.prefix");
+
+  private ByteArrayOutputStream bout;
+  private PrintStream stdOut = System.out;
+  private PrintStream out;
+
+  @Before
+  public void setUp() {
+    bout = new ByteArrayOutputStream();
+    out = new PrintStream(bout);
+    System.setOut(out);
+  }
+
+  @After
+  public void tearDown() {
+    System.setOut(stdOut);
+  }
+
+  @Test
+  public void testChangeStreamSample() {
+    assertNotNull(instanceId);
+    assertNotNull(databaseId);
+    assertNotNull(prefix);
+    ChangeStreamSample.run(instanceId, databaseId, prefix);
+
+    String got = bout.toString();
+    Assert.assertTrue(got.contains("Received a ChildPartitionsRecord"));
+    Assert.assertTrue(got.contains("Received a HeartbeatRecord"));
```

Comment:
Will we always get a heartbeat record? Or can the test be flaky because of that?

Reply:
Removed